src/views/Library.vue

```diff
@@ -12,11 +12,18 @@
     </VRow>
     <VRow v-if="rescan">
       <VBtn @click="start" color="success" class="ma-2">
-        <VIcon left>mdi-refresh {{ rescan.running ? "mdi-spin" : "" }}</VIcon>
+        <VIcon left
+          >mdi-refresh
+          {{ rescan.last_click > rescan.finished_at ? "mdi-spin" : "" }}</VIcon
+        >
```

Comment:
```suggestion
        <VIcon left>
          mdi-refresh
          {{ rescan.last_click > rescan.finished_at ? "mdi-spin" : "" }}
        </VIcon>
```
Linter is being weird.